### ISSUE TITLE
not possible to filter if value have "double-quotes" (Backend)

### ISSUE THREAD

**Issue by zonky2** [contao-community-alliance/dc-general#215]
have value double-quotes like ```Holla the "forest fairy"``` render the widget the follow select option
```
<option value="Holla the "forest fairy"">Holla the "forest fairy"</option>
```

I found two solutions - look at https://github.com/MetaModels/core/issues/925#issuecomment-201016745

**Comment by discordier** [comment]
Imo this should be fixed like the following:
https://github.com/contao-community-alliance/dc-general/blob/83b12c824a14814c3fdabb7340d2bef82b010e11/contao/templates/dcbe_general_panel_filter.html5#L5
```diff
- <option value="<?php echo $arrOption['value']; ?>"<?php echo $arrOption['attributes']; ?>><?php echo $arrOption['content']; ?></option>
+ <option value="<?php echo htmlspecialchars($arrOption['value']); ?>"<?php echo $arrOption['attributes']; ?>><?php echo $arrOption['content']; ?></option>
```

**Comment by stefanheimes** [comment]
+1

**Comment by zonky2** [comment]
fixed with #302
